packages/mobile/src/components/comments/CommentSection.tsx

```diff
@@ -68,18 +68,25 @@ const CommentSectionHeader = (props: CommentSectionHeaderProps) => {
   )
 }
 
-const CommentSectionContent = () => {
+type CommentSectionContentProps = {
+  openCommentDrawer: (autoFocus?: boolean) => void
+}
+
+const CommentSectionContent = (props: CommentSectionContentProps) => {
+  const { openCommentDrawer } = props
   const {
     commentSectionLoading: isLoading,
     commentIds,
     isEntityOwner
   } = useCurrentCommentSection()
 
-  const [postComment] = usePostComment()
+  const handlePress = useCallback(() => {
+    openCommentDrawer()
+  }, [openCommentDrawer])
 
-  const handlePostComment = (message: string, mentions?: ID[]) => {
-    postComment(message, undefined, undefined, mentions)
-  }
+  const handleFormPress = useCallback(() => {
```

Comment:
maybe clearer name to indicate that this is the empty case or a comment about this autofocusing. just to differentiate from above.

Reply:
Agreed, I'm gonna make it a named arg